### PR TITLE
add bigdl.yaml file

### PR DESCRIPTION
**Pull request by zzti-bsj**
1. add `bigdl.yaml` file to support run on k8s

### CODE REVIEW

File: docker/bigdl/bigdl.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
```
Comment (glorysdj) [inline docker/bigdl/bigdl.yaml:1]: rename as deployment.yaml
add howto run to readme

File: docker/bigdl/bigdl.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  replicas: 1
+  selector:
+    matchLabels:
+      app: bigdl-notebook
+  template:
+    metadata:
+      name: bigdl-notebook
+      labels:
+        app: bigdl-notebook
+        appType: mlpNotebook
```
Comment (glorysdj) [inline docker/bigdl/bigdl.yaml:16]: mlpNotebook to bigdl

File: docker/bigdl/bigdl.yaml (hunk 1 above)
Comment (zzti-bsj) [reply docker/bigdl/bigdl.yaml:1]: > rename as deployment.yaml add howto run to readme

ok

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**